corehq/apps/api/resources/v0_5.py

```diff
@@ -1390,3 +1391,42 @@ def get_datasource_data(request, config_id, domain):
     query = cursor_based_query_for_datasource(request_params, datasource_adapter)
     data = response_for_cursor_based_pagination(request, query, request_params, datasource_adapter)
     return JsonResponse(data)
+
+
+class CommCareAnalyticsUserResource(CouchResourceMixin, HqBaseResource, DomainSpecificResourceMixin):
+
+    roles = fields.ListField()
+    permissions = fields.DictField()
+
+    class Meta(CustomResourceMeta):
+        resource_name = 'analytics-roles'
```

Comment:
nit: should we stick to underscore since this would be a part of the URL? I guess its not unusual to have "-" in URLs but is underscore more used.